src/fs_utils.rs

```diff
@@ -202,7 +209,7 @@ mod tests {
         ];
         for (case, expected) in cases.iter() {
             let ek = get_relevant_event_kind(&case);
-            assert_eq!(ek, *expected);
+            assert_eq!(ek, *expected, "case: {:?}", case);
```

Comment:
This was just useful when debugging to actually see the case that failed when the assertion failed.